TACC/Frontera-Portal
# Bug/FP-704

## Overview: ##

The citizenship field in the Required Information form has been removed

## PR Status: ##


## Related Jira tickets: ##

* [FP-704](https://jira.tacc.utexas.edu/browse/FP-704)

## Summary of Changes: ##

Removed the citizenship field

## Testing Steps: ##

In the Manage Account section, open the required information form to see that the field has been removed
